yaraneba/app/controllers/tasks_controller.rb

```diff
@@ -6,7 +6,11 @@ class TasksController < ApplicationController
 
   # GET /tasks or /tasks.json
   def index
-    @tasks = Task.all.order("#{sort_column} #{sort_direction}")
+    @tasks = Task.search(params[:search_title], params[:search_status]).order("#{sort_column} #{sort_direction}")
+    return @tasks if @tasks.blank?
```

Comment:
@tasks を return してもいいですが単にreturn でも大丈夫です。
```suggestion
    return if @tasks.blank?
```